packages/react/src/components/Nav/Nav.base.tsx

```diff
@@ -161,7 +161,10 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
     let finalExpandBtnAriaLabel = '';
     if (link.links && link.links.length > 0) {
       if (link.collapseAriaLabel || link.expandAriaLabel) {
-        finalExpandBtnAriaLabel = link.isExpanded ? link.collapseAriaLabel! : link.expandAriaLabel!;
+        // still respect link.collapseAriaLabel, even though it's deprecated in favor of expandAriaLabel
+        const collapseAriaLabel = link.collapseAriaLabel ?? link.expandAriaLabel;
```

Comment:
I think this is a nit and I'm fine either way but using `||` might be better here as `collpaseAriaLabel` is a string and empty string is falsely but not nullish.

Reply:
I actually think if a team explicitly passes in an empty string, we should respect it. I'm not sure why they would, but maybe something like they want to use `aria-labelledby` instead, or are doing something else strange with the button.

Reply:
Though I could also see it going either way. If there's some use case you're thinking of where authors would set an empty string to the collapse label and expect it to get overridden, I'll change it